### DISCUSSION TITLE
'str' object has no attribute '_meta'

### DISCUSSION THREAD

**Discussion by jskoetsier** [netbox-community/netbox#7505] [Q&A]
Hi,

Since the upgrade from 3.0.4 to 3.0.6 my scripts don't run any more, I get the error:

<class 'AttributeError'>

'str' object has no attribute '_meta'

Python version: 3.8.10
NetBox version: 3.0.6

I think the problem resides in the Meta class, this is my snipplet:

class SyncPrefixes(Script):
    class Meta:
        name = "Synchronize Prefixes from Acceptance netbox"
        description = "This script will synchronize Prefixes from the acceptance netbox"
        commit_default = False

Does anyone have an idea?

**Comment by jeremystretch** [comment]
I'm afraid you've run into #7474. This will be fixed in v3.0.7, which I'm planning to release later today. Sorry for the trouble.

**Reply by jskoetsier** [reply]
Hi, that was a quick reply! Thanks.
